### DISCUSSION TITLE
Connecting on HackBox

### DISCUSSION THREAD

**Discussion by MatthewCalder-msft** [msusdev/Innovation-Challenge#15] [Looking for team members?]
If you are looking to join a team, go to the "Explore Projects" tab and if there is one that looks interesting and is open to join, click "Ask to join" 
![image](https://github.com/user-attachments/assets/493fabd4-63bf-4a88-9900-2aa2ac557900)

If you have a project and are looking to gather your team members, go to the "Find hackers" tab and click "Invite to project" 
![invite to project](https://github.com/user-attachments/assets/29416de6-afea-4126-b3f3-6b5ac90bf35b)

**Comment by screambites** [comment]
@MatthewCalder-msft Hi Matthew, Will we be provided with credits to use Azure services and OpenAI during this challenge. We have exhausted all of ours.

**Reply by MatthewCalder-msft** [reply]
No, I'm sorry we are not able to provide credits. I do encourage you to keep working on project presentations though. You can still get a lot of points in the judging based on the ideas you present.
